flight/mcu/lib/teensyShared/rk4.h

```diff
@@ -0,0 +1,180 @@
+#include <array>
+
+#include "Atmosphere.h"
+
+#pragma once
+
+using std::array;
+
+class rk4 {
+   public:
+    rk4();
+
+    array<float, 2> accel(array<float, 2> u, float rho);
+
+    array<float, 2> rk4_step(array<float, 2> state, float dt, float rho);
+
+    array<float, 2> sim_apogee(array<float, 2> state, float dt);
+
+    float cd(float alt, float vel);
+    array<double, 151> poly = {
```

Comment:
I swear to god @parthshrotri you and your stupid high order polyfits.

Reply:
Also why is this public? I may not be understanding the interaction between kalman, rk4 and activecontrol at all.